src/overlays/actors/ovl_Bg_Mizu_Water/z_bg_mizu_water.h

```diff
@@ -6,10 +6,16 @@
 
 struct BgMizuWater;
 
+typedef void (*BgMizuWaterActionFunc)(struct BgMizuWater*, GlobalContext*);
+
 typedef struct BgMizuWater {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x14];
-} BgMizuWater; // size = 0x0160
+    /* 0x014C */ BgMizuWaterActionFunc actionFunc;
+    /* 0x0150 */ s32 type;
+    /* 0x0154 */ f32 targetY;
+    /* 0x0158 */ f32 baseY;
+    /* 0x015C */ s32 switchFlag; // only used for types 2-4
+} BgMizuWater;                   // size = 0x0160
```

Comment:
just a small thing, we dont run the formatter on headers cause it can do things like this with the comment. if you could move the size comment back to right after the name id appreciate it :D 
everything else looks good to me